common/common_helper.sh

```diff
@@ -264,6 +264,7 @@ function apply_benchmark_load() {
 	if [ ${CLUSTER_TYPE} == "minikube" ]; then
 		TECHEMPOWER_ROUTE=${TECHEMPOWER_URL}
 	elif [ ${CLUSTER_TYPE} == "openshift" ]; then
+ 		oc expose svc/tfb-qrh-service -n ${APP_NAMESPACE}
```

Comment:
@kusumachalasani Please add this in the `get_urls` function where we are exposing the rest of the services

Reply:
done!